bot.py

```diff
@@ -45,11 +45,27 @@ def build_response_text(rate, bank):
     )
         
 def build_response_image(text):
-    photo = Image.new("I", (600, 670), "white")
-    font = ImageFont.truetype("arial.ttf", 25)
-    drawer = ImageDraw.Draw(photo)
-    drawer.text((10, 10), text, font=font, fill='black')
-    return photo 
+    
+    width = 10000
```

Comment:
It is not neccesary to have a separate variables for size of the temporary image